Sources/NIO/Interfaces.swift

```diff
@@ -18,6 +18,8 @@
 //  Created by Cory Benfield on 27/02/2018.
 //
 
+import CNIOLinux
```

Comment:
is this change needed ?

Reply:
For Android - yes, ‘ifaddrs‘ struct defined in CNIOLinux module(ifaddrs-android.h)